config/configload/backend.go

```diff
@@ -186,18 +172,17 @@ func newOAuthBackend(helper *helper, parent hcl.Body) (hcl.Body, error) {
 
 // newTokenRequestBackend prepares a nested backend within each backend-tokenRequest block.
 // TODO: Check a possible circular dependency with given parent backend(s).
-func newTokenRequestBackend(helper *helper, parent hcl.Body) (map[string]hcl.Body, error) {
+func newTokenRequestBackend(helper *helper, parent hcl.Body) (hcl.Body, error) {
```

Comment:
As this now returns the `parent` (the backend with the token requests?), is the function name still appropriate?